admiral/pkg/registry/registry.go

```diff
@@ -0,0 +1,10 @@
+package registry
+
+// IdentityConfiguration is an interface to fetch configuration from a registry
+// backend. The backend can provide an API to give configurations per identity,
+// or if given a cluster name, it will provide the configurations for all
+// the identities present in that cluster.
+type IdentityConfiguration interface {
+	GetByIdentity(identityAlias string) error
```

Comment:
We today use GetByIdentity to get values from cache within admiral codebase, should this be named different may be GetIdentityConfigurationByAlias?

Reply:
Should I rename this to GetByIdentityName?

Reply:
Or, given we are already in the IdentityConfiguration interface, it might be okay if we do GetByIdentity, as its a different function.

And if we add GetConfigurationByIdentity, then we would be repeating the interface name.

Reply:
Sure makes sense, we can use GetByIdentityName, just in case we need another api integration next time for identity